_includes/head.html

```diff
@@ -0,0 +1,12 @@
+<head>
+  <meta charset="utf-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  {%- seo -%}
+  <link rel="stylesheet" href="{{ "/assets/css/style.css" | relative_url }}">
+  {%- feed_meta -%}
+  {%- if jekyll.environment == 'production' and site.google_analytics -%}
+    {%- include google-analytics.html -%}
```

Comment:
What's up with this? Google Analytics?

Reply:
Yes. My understanding is that as current, the EIP repository is using the implicit default default.html file which references the implicit default head.html include. Which is here https://github.com/jekyll/minima/blob/master/_includes/head.html

The file I am checking in matches the implicit default head.html https://github.com/jekyll/minima/blob/master/_includes/head.html and then adds the one MathJaX line.

Reply:
I'm sure it can work without.

Reply:
You are correct.

In the interest of creating a minimal diff, I have not removed that line, which is currently active on the site, and which is currently acting as a no-op.

Reply:
Updated in b48adec43318f7ff14951aa88812ff9db8004943